tests/test_causal_problem/test_callables.py

```diff
@@ -10,47 +10,72 @@
 from causalprog.causal_problem import CausalProblem
 from causalprog.graph import Graph, Node
 
-
-@pytest.fixture
-def n_samples_for_estimands() -> int:
-    return 1000
+n_samples_for_estimands = 1000
```

Comment:
```suggestion
@pytest.fixture
def n_samples_for_estimands() -> int:
 return 1000
```

We shouldn't use a module-scoped variable for this, in the off chance that it gets accidently reassigned during one of the tests.

If you want to avoid a fixture, you can explicitly define it as an argument that takes a constant value to the tests / other fixtures that require it. EG

```python
def expectation_fixture(
    rng_key: jax.Array,
    n_samples_for_estimands: int = 1000,
)
```

Reply:
I changed this while trying to work out why a test was failing and probably shouldn't have committed this bit. Will revert